src/core/components/operation-summary.jsx

```diff
@@ -73,6 +73,7 @@ export default class OperationSummary extends PureComponent {
         {
           (!security || !security.count()) ? null :
             <AuthorizeOperationBtn
+              describedBy={`${tag}-${operationId}`}
```

Comment:
love this! although I think there should maybe be a test that ensures that the id in the operation summary path and the describedBy value here should be the same. maybe by passing it as a prop down somehow. I just worry that having the same thing done in two places might break at some point in the future if someone fiddles with the id in the operation summary path.

Reply:
@leggsimon I worried about the same thing. My ideal solution would actually be a uuid library, but I wanted feedback from project admins before including one in a first PR 😄. Unit tests, at least for this, seem like a good addition though 👍

Reply:
Unit tests would be great, and then I think rather than doing the id attribute creation in both places it might be nice to define the value outside this `return` jsx and pass it as an `id` prop to `OperationSummaryPath` on line 63 maybe?

Reply:
But yes, I would be in favour of a uuid I think. Unfortunately I can’t make those calls either 😄

Reply:
### **Change request**: use path + method instead of tag + operationId here.

Same idea as above (https://github.com/swagger-api/swagger-ui/pull/5298/files#r280963958).

Reply:
Also, a test would be good for the ID - I think it could fit well in either the Enzyme (component-focused) tests or the Cypress (real DOM end-to-end) tests.